card/mod/standard/set/all/rich_html/form.rb

```diff
@@ -227,16 +236,6 @@ def editor_wrap type=nil
     end
   end
 
-  def with_relative_names_in_form
-    @relative_names_in_form = true
-    result = yield
-    @relative_names_in_form = nil
-    result
-  end
-
-  def relative_names_in_form?
-    @relative_names_in_form || (parent && parent.relative_names_in_form?)
-  end
```

Comment:
REMOVED API.

we just always use relative names in forms now.  (relative to the form context).  So this stuff is no longer necessary.